examples/vision/shiftvit.py

```diff
@@ -1,8 +1,8 @@
 """
 Title: A Vision Transformer without Attention
-Author: [Aritra Roy Gosthipaty](https://twitter.com/ariG23498), [Ritwik Raha](https://twitter.com/ritwik_raha)
+Author: [Aritra Roy Gosthipaty](https://twitter.com/ariG23498), [Ritwik Raha](https://twitter.com/ritwik_raha), [Shivalika Singh](https://www.linkedin.com/in/shivalika-singh/)
```

Comment:
@fchollet have updated the author list after checking with @ariG23498 & @ritwikraha 
Hope this is fine.

Reply:
Absolutely fine by me. Keep up the good work!

Reply:
Thank you so much :) @ariG23498